roles/keystone/tasks/domains.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: keystone domain dir
+  file: dest=/etc/keystone/domains state=directory
+
+- name: keystone multi domain conf files
+  template: src="etc/keystone/domains/keystone.{{ item }}.conf"
+            dest="/etc/keystone/domains/keystone.{{ item }}.conf"
+  with_items: keystone.multi_domain.domain_conf_files
+  when: keystone.multi_domain.domain_conf_files.0 is defined
```

Comment:
Yes, perfectly, I intend to parameterize the file once we know what we're changing